### PR TITLE
Setup and add unit tests

### PR DESCRIPTION
**Pull request by tjerman**
Initial tests are lighter since the repo is due for a refactor.

### CODE REVIEW

File: src/views/Bridge/Jitsi.vue [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
-    <span>{{ $t('app.jitsi.createNewRoom') }}</span>
-    <input type="text" id="roomInputField" v-model="roomName" :placeholder="$t('app.jitsi.roomName')" />
-
-    <button :disabled="jitsi || (cleanup(roomName).length === 0)"
-            @click="onCreate">{{ $t('app.jitsi.create') }}</button>
-
-    <div ref="jitsiInterface"
-         v-show="jitsi"
-         class="jitsiInterface"></div>
+    <p v-if="jistsiScriptErr || promiseError" class="error">{{ jistsiScriptErr || promiseError }}</p>
```
Comment (darh) [inline src/views/Bridge/Jitsi.vue]: Why are we changing this (in this PR)?

File: src/components/AppSelectorItem.vue [hunk 2]
```diff
@@ -18,17 +18,17 @@
...
         class='app-text'>
-        {{ crustapp.name }}
+        {{ appName }}
       </label>
     </div>
     <!-- this should appear on hover -->
     <div class="app-display-options">
-      <div class="display-option"><a class="link" @click="$emit('doadd')">{{ $t('app.general.openTab') }}</a></div>
-      <div class="display-option"><a class="link" target="_blank" :href="crustapp.url">{{ $t('app.general.openWindow') }}</a></div>
+      <div class="display-option open-tab"><a class="link" @click="$emit('doadd')">{{ $t('app.general.openTab') }}</a></div>
```
Comment (darh) [inline src/components/AppSelectorItem.vue]: why are we changing this here?

Comment (tjerman) [reply src/components/AppSelectorItem.vue]: For this test https://github.com/cortezaproject/corteza-webapp-unify/pull/31/files#diff-6551d6c87be5b8260ad9742e4f5543d2R16

File: src/views/Bridge/Jitsi.vue (hunk 1 above)
Comment (tjerman) [reply src/views/Bridge/Jitsi.vue]: It seemed convenient to add this since I tweaked err handling after because of testing.

**Pull request closed without merging**